ONSdigital/census-rm-performance-tests
# Adding initial contact performance test

# Motivation and Context
To test the capability of our services, we've created a performance test which looks at loading a sample and producing print files for initial contact.  The test then prints out how long it takes to produce print files after the action rule is triggered.

# What has changed
- Added a performance test
- Able to push to kubernetes and test 

# How to test?
- Run the performance test with only 1 per treatment code and the test should go through fine. 
- Run in kubernetes.
- Run a performance test with 1000 per treatment code and `ACTION_RULE_DELAY_MINUTES` set to about 10 mins.

# Links
[Trello](https://trello.com/c/CdeYNLlj)
